src/components/EmojiPicker/EmojiPickerMenu/index.native.js

```diff
@@ -68,6 +82,18 @@ class EmojiPickerMenu extends Component {
         return this.props.windowWidth >= this.props.windowHeight;
     }
 
+    /**
+     * Update user preferred skin tone
+     *
+     * @param {Number} skinTone
+     */
+    updatePreferredSkinTone(skinTone) {
+        if (this.props.preferredSkinTone === skinTone) {
+            return;
+        }
+
+        this.props.updatePreferredSkinTone(skinTone);
+    }
```

Comment:
This should be passed on `EmojiSkinToneList` instead of `this.props.updatePreferredSkinTone`

```
<EmojiSkinToneList
                    updatePreferredSkinTone={this.props.updatePreferredSkinTone}
                    preferredSkinTone={this.props.preferredSkinTone}
                />
```